collections/nemo_nlp/nemo_nlp/utils/callbacks/ner.py

```diff
@@ -92,7 +98,7 @@ def eval_epochs_done_callback(global_vars, tag_ids, output_filename):
     r = correct_chunks / total_chunks if total_chunks > 0 else 0
     f1 = 2 * p * r / (p + r) if p > 0 and r > 0 else 0
 
-    print(f"Accuracy = {accuracy}")
-    print(f"F1 = {f1}")
+    logger.info(f"Accuracy = {accuracy}")
```

Comment:
Does this work with logger from ExpManager?

Reply:
yes